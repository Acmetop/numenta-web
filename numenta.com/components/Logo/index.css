/**
 * Numenta shared web components
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.logo {
  composes: ma0 from 'tachyons';
  composes: pa0 from 'tachyons';
}
