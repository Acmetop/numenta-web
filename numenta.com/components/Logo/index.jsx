import React from 'react'

import Image from '../Image'

import logoFile from './images/numenta.svg'
import styles from './style.css'


const Logo = () => (
  <Image
    alt="Numenta Logo"
    className={styles.logo}
    shadow={false}
    src={logoFile}
    title="Numenta"
  />
)

export default Logo
