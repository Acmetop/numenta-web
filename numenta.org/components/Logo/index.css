/**
 * Numenta.org HTM Community website source code
 * MIT License (see LICENSE.txt)
 * Copyright © 2005—2016 Numenta <http://numenta.com>
 */

.logo {
  composes: ma0 from 'tachyons';
  composes: pa0 from 'tachyons';
}
