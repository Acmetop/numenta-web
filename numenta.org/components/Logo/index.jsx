// Numenta.org HTM Community website source code
// MIT License (see LICENSE.txt)
// Copyright © 2005—2016 Numenta <http://numenta.com>

import React from 'react'

import Image from '../Image'

import logoFile from './images/numenta.svg'
import styles from './index.css'


const Logo = () => (
  <Image
    alt="Numenta Logo"
    className={styles.logo}
    shadow={false}
    src={logoFile}
    title="Numenta"
  />
)

export default Logo
