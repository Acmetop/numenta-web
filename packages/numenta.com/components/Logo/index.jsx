// Numenta shared web components
// MIT License (see LICENSE.txt)
// Copyright © 2005—2016 Numenta <http://numenta.com>

import React from 'react'

import Image from 'numenta-web-shared-components/Image'

import logoFile from './images/numenta.svg'
import styles from './index.css'


/**
 * Site Logo image - React view component.
 */
const Logo = () => (
  <Image
    alt="Numenta Logo"
    className={styles.logo}
    shadow={false}
    src={logoFile}
    title="Numenta"
  />
)

export default Logo
