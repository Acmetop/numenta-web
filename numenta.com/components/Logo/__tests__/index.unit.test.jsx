import React from 'react'
import renderer from 'react-test-renderer'

import Logo from '../../Logo'


describe('<Logo /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <Logo />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
