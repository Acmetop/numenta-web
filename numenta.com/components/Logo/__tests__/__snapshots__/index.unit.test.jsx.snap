exports[`<Logo /> React component unit test suite Renders correctly 1`] = `
<img
  alt="Numenta Logo"
  className="image"
  onClick={undefined}
  src="mockfile.ext"
  title="Numenta" />
`;
